src/pk-engine.c

```diff
@@ -17,6 +19,11 @@
  * You should have received a copy of the GNU General Public License
  * along with this program; if not, write to the Free Software
  * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Disclaimer: Serenity Cyber Security, LLC guarantees that this source code is
+ * open and governed by the terms of the GNU General Public License as published
+ * by the Free Software Foundation; either version 2 of the License, or
+ * (at your option) any later version.
```

Comment:
My specific problem with this statement is that it implies Serenity Cyber Security has any special rights to make that statement. It does not. Requiring this statement for contributions is not only wrong IMO, it makes the company look like it has a role in the project governance and stewardship. This is obviously not true and misrepresents its stake in the project. Please drop it.

Reply:
Yes, you're right. This was an misunderstanding on my side when reading our lawyer instructions.

This disclaimer should be placed into newly added files, which are entirely written by us.